BLSQ/iaso
# IA-3101:Fill the "location field" in the form, so that it can be used, e.g. when a form is created in ENKETO

Explain what problem this PR is resolving

Related JIRA tickets : [IA-3101](https://bluesquare.atlassian.net/browse/IA-3101)

## Self proofreading checklist


## Changes

Renamed the location field with the same name as in the xls form and filled it with the GPS of org unit.

## How to test

Launch setuper to generate data and open forms list page : [http://localhost:8081/dashboard/forms/list](http://localhost:8081/dashboard/forms/list).

From the forms page, open the form **Data for Health facility/Données Formation sanitaire** setting form and display its **advanced settings** , you should see in location field **coordonnees_gps_fosa**.

Then go back to submissions linked to the form, open any submission in enketto. You should see also the GPS linked to the submission.


## Print screen / video

https://github.com/BLSQ/iaso/assets/6383219/361836fc-fef2-48b4-8ea7-71914175b07c


[IA-3101]: https://bluesquare.atlassian.net/browse/IA-3101?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
